pages/linux/xfce4-screenshooter.md

```diff
@@ -0,0 +1,31 @@
+# xfce4-screenshooter
+
+> The XFCE4 screenshot tool.
+
+- Launch the screenshooter GUI:
+
+`xfce4-screenshooter`
+
+- Take a screenshot of the entire screen and launch the GUI to ask how to proceed:
```

Comment:
How about?
```suggestion
- Take a screenshot of the entire screen and save it interactively:
```

Reply:
Hm, I wouldn't say "save it". The GUI offers various options apart from saving: copy to clipboard, discard it, upload to the web, etc.

Reply:
I guess if you wanted to use that turn of phrase, you'd be after something like `process it` instead of save it. Personally, I think it's probably ok as-is (though perhaps `a GUI` would read slightly better?)